lib/src/main/java/io/ably/lib/http/TokenAuth.java

```diff
@@ -36,11 +36,11 @@ public void setTokenDetails(TokenDetails tokenDetails) {
 		this.encodedToken = Base64Coder.encodeString(tokenDetails.token).replace("=", "");
 	}
 
-	public TokenDetails authorise(AuthOptions options, TokenParams params, boolean force) throws AblyException {
+	public TokenDetails authorise(AuthOptions options, TokenParams params) throws AblyException {
 		Log.i("TokenAuth.authorise()", "");
 		if(tokenDetails != null) {
 			if(tokenDetails.expires == 0 || tokenValid(tokenDetails)) {
-				if(!force) {
+				if(!options.force) {
```

Comment:
I thought about that too, and decided not to take an action since test cases under `RestAuthTest` didn't failed. Now that, Travis reported `RealtimeConnectFailTest`'s `connect_token_expire_disconnected` failing because of that reason, I will fix it now